media/js/privacy/redirects.es6.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+if (window.location.hash.indexOf('#health-report') === 0) {
```

Comment:
Needs to account for the automation query string (ie. `http://localhost:8000/en-US/privacy/firefox/#health-report?automation=true`)